sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/OperationStatus.java

```diff
@@ -11,6 +11,13 @@
 /** Known values for operation status. */
 @Immutable
 public final class OperationStatus extends ExpandableStringEnum<OperationStatus> {
+
+    /**
+     * Creates a OperationStatus object.
+     */
+    public OperationStatus() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.